NEWS.md

```diff
@@ -39,6 +39,9 @@
 - Note to packagers: The manual pages have been moved to resources/man
   and now include man pages for mkosi-initrd and mkosi-sandbox as
   well.
+- `InitrdInclude=` was removed. If you're using `InitrdInclude=`, please
+  build your initrd via a subimage in `mkosi.images` containing
+  `Include=mkosi-initrd` and any customizations you wish to add.
```

Comment:
How exactly should one use that? Naming the image initrd and adding the include does not seem to work. I could add a postbuild step to move the generated initrd to `$ARTIFACTDIR/io.mkosi.initrd` but that seems like a hacky solution. Am I missing an easier solution?

Reply:
`Initrds=` setting

Reply:
Ah yeah I forgot about that...I knew I was missing sth